CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this
 file. This file is structured according to http://keepachangelog.com/
 
 - - -
+## 4.2.1 - 2017-01-31
```

Comment:
Wait you are not changing the code here, just the tests, so nothing should change in the release notes, right?